src/zephyr_coap_req.c

```diff
@@ -697,13 +697,18 @@ static int64_t golioth_coap_req_poll_prepare(struct golioth_coap_req *req, uint3
 
     if (send)
     {
+#ifndef CONFIG_LOG_BACKEND_GOLIOTH
```

Comment:
Is there a way we can log this to all backends except for Golioth, rather than not logging it at all?